### PR TITLE
Fixed wheel generation

### PR DESCRIPTION
**Pull request by lajerm**

### CODE REVIEW

File: scripts/install-deps.sh [hunk 1]
```diff
@@ -2,29 +2,47 @@
...
-    openmpi-bin \
-    openmpi-doc \
-    libopenmpi-dev \
-    libopenblas-dev \
-    libarpack2-dev \
-    libparpack2-dev
+platform="$(uname)"
+
+if [ "${platform}" == "Linux" ]; then
+    # sudo apt-get update -y
```
Comment (mpmdean) [inline scripts/install-deps.sh:8]:
```suggestion
    sudo apt-get update -y
```

File: scripts/install-deps.sh [hunk 2]
```diff
@@ -2,29 +2,47 @@
...
-    gfortran \
-    openmpi-bin \
-    openmpi-doc \
-    libopenmpi-dev \
-    libopenblas-dev \
-    libarpack2-dev \
-    libparpack2-dev
+platform="$(uname)"
+
+if [ "${platform}" == "Linux" ]; then
```
Comment (mpmdean) [inline scripts/install-deps.sh:7]: As I checked in #217 it seems that commenting these install is what is breaking the generations of the docs.

I know that we wanted to install these via conda, but I don't think that is properly implemented?

File: scripts/install-deps.sh [hunk 3]
```diff
@@ -2,29 +2,47 @@
...
+    # sudo apt-get update -y
+    # sudo apt-get install -y \
```
Comment (mpmdean) [inline scripts/install-deps.sh:9]:
```suggestion
    sudo apt-get install -y \
```

File: scripts/install-deps.sh [hunk 4]
```diff
@@ -2,29 +2,47 @@
...
+    #     gfortran \
```
Comment (mpmdean) [inline scripts/install-deps.sh:10]:
```suggestion
        gfortran \
```

File: scripts/install-deps.sh [hunk 5]
```diff
@@ -2,29 +2,47 @@
...
+    #     openmpi-bin \
```
Comment (mpmdean) [inline scripts/install-deps.sh:11]:
```suggestion
        openmpi-bin \
```

File: scripts/install-deps.sh [hunk 6]
```diff
@@ -2,29 +2,47 @@
...
+    #     openmpi-doc \
```
Comment (mpmdean) [inline scripts/install-deps.sh:12]:
```suggestion
       openmpi-doc \
```

File: scripts/install-deps.sh [hunk 7]
```diff
@@ -2,29 +2,47 @@
...
+    #     libopenmpi-dev \
```
Comment (mpmdean) [inline scripts/install-deps.sh:13]:
```suggestion
        libopenmpi-dev \
```

File: scripts/install-deps.sh [hunk 8]
```diff
@@ -2,29 +2,47 @@
...
+    #     libopenblas-dev \
```
Comment (mpmdean) [inline scripts/install-deps.sh:14]:
```suggestion
       libopenblas-dev \
```

File: scripts/install-deps.sh [hunk 9]
```diff
@@ -2,29 +2,47 @@
...
+    #     libarpack2-dev \
```
Comment (mpmdean) [inline scripts/install-deps.sh:15]:
```suggestion
        libarpack2-dev \
```

File: scripts/install-deps.sh [hunk 10]
```diff
@@ -2,29 +2,47 @@
...
+    #     libparpack2-dev
```
Comment (mpmdean) [inline scripts/install-deps.sh:16]:
```suggestion
        libparpack2-dev
```

File: scripts/install-deps.sh [hunk 11]
```diff
@@ -2,29 +2,47 @@
...
+    echo "Skipping system packages installation in favor of conda packages for building."
```
Comment (mpmdean) [inline scripts/install-deps.sh:17]:
```suggestion
    # install these packages via conda in the future
```

**Review by mpmdean — COMMENTED** [review]
I beleive that it is premature to remove all these apt-get installs, as I don't think installing these via conda is working see #217 , which seems to solve the currently failing docs build.

**Review by mrakitin — COMMENTED** [review]
I think it's ready for review. 

@mpmdean, if the CI runs pass - feel free to merge! We'll see if there are any issues with publishing docs as they are published only from the `master` branch.

**Comment by mrakitin** [comment]
I pushed a few commits to fix the docs artifact uploading/downloading business. Now it works:
https://nsls-ii.github.io/edrixs/reference/index.html
image

**Pull request merged**